design/ui/pagination/src/Pagination.test.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { cleanup, fireEvent, render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+/**
+ * We import the fixture to save us from the setup hassle,
+ * the fixture contains everything we need for testing
+ * the component */
+import { Wrapper } from './Pagination.fixture';
+
+describe('Espressive UI Pagination', () => {
+  afterEach(cleanup);
```

Comment:
We also do not have to do `afterEach(cleanup)` anymore in our Jest tests. We have it configured to automatically do this now. 🎉 No need to change here for now either.